lib/ship.rb

```diff
@@ -3,13 +3,12 @@
 
 class Ship
 
-  attr_reader :ship_name, :length
-  attr_accessor :health
+  attr_reader :name, :length, :health
```

Comment:
Even though `attr_accessor` worked it seems to be best practice to use `attr_reader` when possible.